src/features/auth/components/AuthenticationButton/AuthenticationButton.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import React from 'react'
+
+import { AuthenticationButton } from './AuthenticationButton'
+
+export default {
+  title: 'Features/Auth/LogInButton',
+  component: AuthenticationButton,
+} as ComponentMeta<typeof AuthenticationButton>
+
+const Template: ComponentStory<typeof AuthenticationButton> = (props) => (
+  <AuthenticationButton {...props} />
+)
+
+// TODO(PC-17931): Fix those stories
+const Login = Template.bind({})
+Login.storyName = 'LoginButton'
+Login.args = {
+  type: 'login',
+}
```

Comment:
Tu n'es pas obligé de mettre le storyName, tu peux juste faire ça : 

```suggestion
const Login = Template.bind({})
Login.args = {
  type: 'login',
}
```